src/base/snark0.ml

```diff
@@ -714,7 +715,7 @@ module Run = struct
 
   module Make_basic (Backend : Backend_intf.S) = struct
     module Snark = Make (Backend)
-    open Run_state
+    open Backend.Run_state
```

Comment:
It seems we can remove this open by changing L1564. See https://github.com/o1-labs/snarky/commit/242d5b55a169f93c1540ec75a3bfc90c81c4cbac.